srcpkgs/sandboxfs/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'sandboxfs'
+pkgname=sandboxfs
+version=0.1.0
+revision=1
+wrksrc="sandboxfs-sandboxfs-${version}"
+archs="noarch"
```

Comment:
really? I'd think this is wrong.

Reply:
The template is still being created incorrectly, as is addressed in another PR. I neglected to xlint the template this time.  Will fix.

Reply:
and removing the noarch statement